docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -948,7 +946,7 @@ def get_stringifiable_version(label):
   return 'str(' + label + ')'
 
 def get_reserved_label_parts(prefixes, label):
-  return re.search(rf'{prefixes}(\d*)(.*)', label)
+   return re.search(fr"{prefixes}(\d*)(.*)", label)
```

Comment:
Would be interested in understanding the reason for this change in the order. Is it functional or aesthetic?

Reply:
I was trying to solve a linter error message. Turns out I had my Python interpreter on 2.7 before format strings were supported. Not a functional change, but this just matches the documentation example I found so I tried it to resolve the error before realizing it was Py version issue.